tests_e2e/tests/agent_update/rsm_update.py

```diff
@@ -0,0 +1,143 @@
+#!/usr/bin/env python3
+
+# Microsoft Azure Linux Agent
+#
+# Copyright 2018 Microsoft Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+#
+# BVT for the agent update scenario
+#
+import json
+import os.path
+import time
+
+import requests
+from assertpy import assert_that
+from azure.identity import DefaultAzureCredential
+from azure.mgmt.compute.models import VirtualMachine
+
+from tests_e2e.tests.lib.agent_test import AgentTest
+from tests_e2e.tests.lib.agent_test_context import AgentTestContext
+from tests_e2e.tests.lib.logging import log
+from tests_e2e.tests.lib.ssh_client import SshClient
+from tests_e2e.tests.lib.virtual_machine import VmMachine
+
+
+class RsmUpdateBvt(AgentTest):
+
+    def __init__(self, context: AgentTestContext):
+        super().__init__(context)
+        self._ssh_client = None
+
+    def run(self):
+        self._ssh_client = SshClient(
+            ip_address=self._context.vm_ip_address,
+            username=self._context.username,
+            private_key_file=self._context.private_key_file)
+
+        # Allow agent to send supported feature flag
+        time.sleep(60)
+
+        log.info("*******Verifying the Agent Downgrade scenario*******")
+        self._prepare_rsm_update("1.3.0.0")
```

Comment:
Are we going to delete 1.3.{0/1}.0 versions or keep them published for these tests?

Reply:
we don't publish again, and it's already published. We keep using them for this scenario.